src/components/collection-progress-group/components/collection-progress-status-section/collection-progress-status-section.module.css

```diff
@@ -10,6 +10,10 @@
 	display: flex;
 	gap: 8px;
 	width: 100%;
+
+	/* Height of the button link component */
+	padding-top: 5px;
+	padding-bottom: 5px;
```

Comment:
This css isn't being applied for mobile, tablet or desktop, so let's cut it.
```suggestion
```
<img width="150" alt="Screenshot 2025-01-16 at 11 32 06" src="https://github.com/user-attachments/assets/6230d912-6b5b-4e37-91ab-682935fd23f4" />

Reply:
Oh that's weird. Without the CSS, the padding looks off (see the second screenshot in the PR description). I have a feeling that the button expands the parent container to expand vertically.

Reply:
My bad, the parent component has the same css because of [this line](https://github.com/hashicorp/dev-portal/pull/2665/files#diff-90da2f7b6b2691b855b3b1f7c67b60370558c568c53cba1b5103feff4fbb73f8R21). I was looking at the css for that element, not the child element.